extras/data/machinery/generate/EMC.py

```diff
@@ -0,0 +1,40 @@
+from .utils import load, obs_template
+
+
+def dump(src_path, target):
+    """Compute SLAC observables.
+
+    Parameters
+    ----------
+    src_path : str
+        input path
+
+    Returns
+    -------
+    dict
+        observables dictionary, corresponding to the runcard
+
+    """
+    del target
+
+    obs = obs_template.copy()
+    data = load(src_path, 0, ["-", "x", "Q2", "y"])
+    esf = [dict(x=d["x"], y=d["y"], Q2=d["Q2"]) for d in data]
```

Comment:
you can set a `y` here (and I believe you even have to, due to the interface), but of course it will be completely ignored (because $F_2 = F_2(x,Q^2)$ )

Reply:
Could you expand on what you mean by this `I believe you even have to, due to the interface`? I know that this will be skipped in `esf` but wanted to dump them anyway just to be consistent with the data tables.